packages/turf-along/test/fixtures/dc-points.geojson

```diff
@@ -1,115 +1,93 @@
 {
-  "type": "FeatureCollection",
-  "features": [
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.0316696166992,
-          38.87406218243845
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.02325820922852,
-          38.885688179036094
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.0222282409668,
-          38.89744587262311
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.02377319335936,
-          38.910804525446686
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.02840805053711,
-          38.91441093075183
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.02840805053711,
-          38.92402711565758
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.04008102416992,
-          38.932707274379595
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -76.99390411376953,
-          38.91387666004744
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.03269958496094,
-          38.898648254305215
-        ]
-      }
-    },
-    {
-      "type": "Feature",
-      "properties": {},
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          -77.02342987060545,
-          38.870587377511235
-        ]
-      }
-    }
-  ]
-}
\ No newline at end of file
+	"type": "FeatureCollection",
```

Comment:
not sure what's the style on `JSON` indentation, 2 or 4 spaces?

Reply:
dont think we've ever cared have we?